index.html

```diff
@@ -54,58 +55,351 @@ <h1> Space Ripples Reveal Big Bang’s Smoking Gun</h1>
     </div>
   </header>
 
-
   <main>
-    <div id='main-body'>
-      <div class='author'>
-        <h3><a rel="noopener" target='_blank' href="https://www.nytimes.com/by/dennis-overbye">By Dennis Overbye</a></h3>
+    <div id='main-body' class='flex'>
+      <h2><a rel="noopener" target='_blank' href="https://www.nytimes.com/by/dennis-overbye">By Dennis Overbye</a></h2>
+      <p> March 17, 2014</p>
+      <p>CAMBRIDGE, Mass. — One night late in 1979, an itinerant young physicist named Alan Guth, with a new son and a year’s appointment at Stanford, stayed up late with his notebook and equations, venturing far beyond the world of known physics.
+      </p>
+      <p>He was trying to understand why there was no trace of some exotic particles that should have been created in the Big Bang. Instead he discovered what might have made the universe bang to begin with. A potential hitch in the presumed course
+        of cosmic evolution could have infused space itself with a special energy that exerted a repulsive force, causing the universe to swell faster than the speed of light for a prodigiously violent instant.</p>
+      <p>If true, the rapid engorgement would solve paradoxes like why the heavens look uniform from pole to pole and not like a jagged, warped mess. The enormous ballooning would iron out all the wrinkles and irregularities. Those particles were not
+        missing, but would be diluted beyond detection, like spit in the ocean.</p>
+      <p>“SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page and drew a double box around it.</p>
+      <p>On Monday, Dr. Guth’s starship came in. Radio astronomers reported that they had seen the beginning of the Big Bang, and that his hypothesis, known undramatically as inflation, looked right.</p>
+      <p>Reaching back across 13.8 billion years to the first sliver of cosmic time with telescopes at the South Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian Center for Astrophysics detected ripples in the fabric of
```

Comment:
## Implement the `Editor's pick` Section
- [ ]  The `editor's pick` section is currently missing. Let's implement it, please.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/amiraabouhadid/project1/page-styling/index.html)|
|:--:|:--:|
|![editor's-pick](https://user-images.githubusercontent.com/22345961/108413508-2a681300-723c-11eb-8cc0-7ffc2be5e18f.png)|![editor's-pick-clone](https://user-images.githubusercontent.com/22345961/108413525-2fc55d80-723c-11eb-9d6d-8ef30cc55ba4.png)|

Reply:
why is it showing it at the bottom of the page here then?

![Screenshot from 2021-02-18 23-03-59](https://user-images.githubusercontent.com/56790126/108421342-baf32300-723d-11eb-801f-babcfdc0270a.png)